tools/xtb/.shed.yml

```diff
@@ -0,0 +1,12 @@
+owner: recetox
```

Comment:
Such a kind of shed.yml https://github.com/galaxyproject/tools-iuc/blob/main/tools/aldex2/.shed.yml

is enough. You don't need the include part etc ...

Reply:
Updated (4d60285)